### PR TITLE
adjust jwt convertation

### PR DESCRIPTION
**Pull request by andrei-astaschenko**

### CODE REVIEW

File: src/main/java/com/epam/petclinic/order/config/ResourceServerConfiguration.java [hunk 1]
```diff
@@ -1,30 +1,35 @@
...
+            // example of adding request security
+            .antMatchers(HttpMethod.GET, "/orders").authenticated()
+            .and().authorizeRequests()
+            .antMatchers(HttpMethod.GET, "/orders/clinic-test").hasAuthority("admin")
+            .anyRequest().authenticated();
     }
+
     @Override
     public void configure(ResourceServerSecurityConfigurer resources) {
-        resources.resourceId("ORDER-SERVICE").tokenStore(tokenStore);
```
Comment (Andrei-Khadziukou) [inline src/main/java/com/epam/petclinic/order/config/ResourceServerConfiguration.java:28]: How it will work if we need to connect to one more service?

Comment (andrei-astaschenko) [reply src/main/java/com/epam/petclinic/order/config/ResourceServerConfiguration.java:28]: Client token contains encoded information about allowed resourceIds. It means that if request with token wants to get access to resource-server, It decodes token and checks allowed resourceIds.

Comment (Andrei-Khadziukou) [reply src/main/java/com/epam/petclinic/order/config/ResourceServerConfiguration.java:28]: Well... I want to know how I can add token for the another service if there is any... For instance, I want to connect to ANIMAL-SERVICE, how I can specify the token for it?

Comment (Andrei-Khadziukou) [reply src/main/java/com/epam/petclinic/order/config/ResourceServerConfiguration.java:28]: Solved, specified public key is used to decode token comiing from Authentication server (there are principle, permissions and other importnat information to reduce number of calls to AUTH-SERVER). Add comment to key in property file

**Pull request merged**